ui-admin/src/navbar/StudySidebar.tsx

```diff
@@ -78,10 +78,6 @@ export const StudySidebar = ({ study, portalList, portalShortcode }:
           <NavLink to={studyEnvNotificationsPath(portalShortcode, study.shortcode, 'sandbox')}
             className={sidebarNavLinkClasses} style={navStyleFunc}>Emails &amp; Notifications</NavLink>
         </li>
-        <li>
-          <NavLink to={studyEnvAlertsPath(portalShortcode, study.shortcode, 'sandbox')}
-            className={sidebarNavLinkClasses} style={navStyleFunc}>Participant Dashboard</NavLink>
-        </li>
```

Comment:
We're deprecating custom dashboard messages for the moment. All of the underlying code is still there at https://localhost:3000/demo/studies/heartdemo/env/sandbox/alerts

Also open to removing it all entirely, but didn't want to be lump too much into this PR